_episodes/#01-basics.md#

```diff
@@ -0,0 +1,57 @@
+---
+title: Basics of Testing
+teaching: 5
+exercises: 0
+questions:
+- "Why would you not want to test?"
+objectives:
+- "Understand the place of testing in a scientific workflow."
+- "Understand that testing has many forms."
+keypoints:
+- "Tests compare that the result observed from running code is the same as what was expected ahead of time."
+- "Tests should be written at the same time as the code they are testing is written."
```

Comment:
I don't necessarily think this is true. Tests are, arguably, more effective if written beforehand. And, many tests must, by their nature, be written after. Where is this statement coming from?